javascript/src/async/variables.js

```diff
@@ -154,6 +163,12 @@ export default store => {
         }
     })
 
+    resync(store, climateAndRegionSelect, (state, io, dispatch, previousState) => {
```

Comment:
You can leave off `previousState` form the arguments, since you're not using it.